### ISSUE TITLE
I have two questions

### ISSUE THREAD

**Issue by smmsamm** [zalo/LaplASCIIan#9]
I really liked your project, I have two questions, do you have a chance to explain to a newbie:
1- How can I run it locally with my own characters and font?
2- Can I get GIF or MP4 animation output directly from it?

**Comment by zalo** [comment]
Yeah, you basically just want to edit the Python file to call this function yourself:
https://github.com/zalo/LaplASCIIan/blob/master/api/index.py#L25

You can see where it loads Consolas.ttf in there too, just swap that out.

.gif or .mp4 output is a bit trickier… theoretically there is an intermediate stage where a rasterized version of the frame sooort of exists, but it’d probably be easier just to record your screen after the fact…

**Comment by smmsamm** [comment]
**thank you
when I click on index.py in command I have:**
 * Serving Flask app 'index'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://localhost:8080
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 816-846-614
 * 
 **but when i use http://localhost:8080 in my browser it shows me:**
 
Method Not Allowed
The method is not allowed for the requested URL.

**Comment by smmsamm** [comment]
ok after some googling;
I downloded python and installed all needed from requirements.txt
then used xampp for make it offline and used link:
http://localhost/LaplASCIIan/
you know there is an input link for give the software my local animation gif in "URL:"
I found a good animation gif in link:
https://i.giphy.com/media/rHkyeLUQn2A6zm87zU/giphy.gif
download it to directory:
F:\xampp\htdocs\laplasciian

now when I use "https://i.giphy.com/media/rHkyeLUQn2A6zm87zU/giphy.gif" for input gif,
**Application do in correct mode** without error but if i use:
http://localhost/LaplASCIIan/giphy.gif
or
./giphy.gif
or 
file:///F:/xampp/htdocs/laplasciian/giphy.gif
it shows error:
"Conversion Failed; Error (None): None"
or even if i put:
"giphy.gif"
it shows error:
Something went wrong with loading the gif...

please help to solve the problem
or help me to run python code on my local gifs
thank you

**Comment by zalo** [comment]
[On this line, it tries to open it like a URL](https://github.com/zalo/LaplASCIIan/blob/a9d57c72c3c7f0b52b7cc3cd536de8243702e8dc/api/index.py#L16)

You probably want to replace 
```python
imageio.mimread(imageio.core.urlopen(gifURL).read(), '.gif')
``` 
with something like:
```python
imageio.mimread(gifURL, '.gif')
```
or this if you just want to hard-code it:
```python
imageio.mimread("F:\xampp\htdocs\laplasciian\giphy.gif", '.gif')
```
